features/admin-file-attributes.feature

```diff
@@ -0,0 +1,99 @@
+@admin-file @issue-457
+  Feature: A curator can manage file attributes in admin file update page
+    As a curator,
+    I want to manage file attributes from the update form
+    So that I can associate various attributes to files
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+    And an admin user exists
+
+  @ok
+  Scenario: Guest user cannot visit admin file update page
+    Given I am not logged in to Gigadb web site
+    When I go to "/adminFile/update/"
+    Then I should see "Login"
+
+  @ok
+  Scenario: Go to a published dataset found in production-like database
+    Given I am not logged in to Gigadb web site
+    When I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    Then I should see "History" tab with text "File Termitomyces_assembly_v1.0.fa.gz updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+
+  @ok @Published
+  Scenario: Sign in as admin and visit admin file update page and see New Attribute, Edit, Delete, Save buttons
+    Given I sign in as an admin
+    When I am on "/adminFile/update/id/13973"
+    Then I should see a button "New Attribute"
+    And I should see a file attribute table
+      | Attribute Name | Value | Unit |
+      | last_modified  | 2013-7-15 |  |
+    And I should see a button input "Edit"
+    And I should see a button input "Delete"
+    And I should see a button input "Save"
+
+  @ok @javascript @Published
+  Scenario: Sign in as admin, delete an attribute of a published dataset and save, then check for history tab
+    Given I sign in as an admin
+    And I go to "/adminFile/view/id/13973"
+    And I should see field "File Attribute Id" with value "1 "
+    When  I go to "/adminFile/update/id/13973"
+    And I should see a file attribute table
+    | Attribute Name | Value | Unit |
+    | last_modified  | 2013-7-15 |  |
+    And I should see a button input "Delete"
+    And I press "Delete"
+    And I should not see "last_modified"
+    And I should not see "2013-7-15"
+    And I should not see a button "Delete"
+    And I press "Save"
+    Then I go to "/adminFile/view/id/13973"
+    And I should see field "File Attribute Id" without value "1 "
+    And I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    And I should see "History" tab with text "Termitomyces_assembly_v1.0.fa.gz: file attribute deleted"
```

Comment:
@kencho51 I think its better to break this scenario into smaller scenarios because it appears there are observed outcomes in `Given` steps and in When` steps. For example, `And I should see field "File Attribute Id" with value "1 "` sounds like an observed outcome but it is in a `Given` step. Outcomes to be observed should be in `Then` steps. Take a look at scenarios in other features and the behat [documentation](https://docs.behat.org/en/latest/user_guide/writing_scenarios.html).